docs/quickstart_guide.md

```diff
@@ -130,6 +133,13 @@ source /etc/profile && go build && sudo ./vhive
 ./scripts/github_runner/clean_cri_runner.sh
```

Comment:
It cannot be in the same snippet, people might execute it blindly, removing the cluster.

Reply:
It is not in the same snippet.
![image](https://user-images.githubusercontent.com/20225226/114759204-7edec900-9d7b-11eb-88f0-ab76a8903287.png)